projects/packages/my-jetpack/_inc/components/product-interstitial/index.jsx

```diff
@@ -28,10 +28,14 @@ export default function ProductInterstitial( { slug, children = null } ) {
 	useEffect( () => {
 		recordEvent( 'jetpack_myjetpack_product_interstitial_view', { product: slug } );
 	}, [ recordEvent, slug ] );
+
+	const trackProductClick = useCallback( () => {
+		recordEvent( 'jetpack_myjetpack_product_interstitial_add_link_click', { product: slug } );
+	}, [ recordEvent, slug ] );
 	return (
 		<Container className={ styles.container } horizontalSpacing={ 0 } horizontalGap={ 0 }>
 			<Col sm={ 4 } md={ 4 } lg={ 5 }>
-				<ProductDetail slug={ slug } />
+				<ProductDetail slug={ slug } trackButtonClick={ trackProductClick } />
```

Comment:
I think the event should be named `onAdd()`, or something like that. If you agree, we can rename it in a follow-up PR

```suggestion
				<ProductDetail slug={ slug } onAdd={ trackProductClick } />
```